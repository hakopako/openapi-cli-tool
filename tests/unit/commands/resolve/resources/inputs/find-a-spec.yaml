paths:
  /pets:
    post:
      summary: Add a new pet
      requestBody:
        description: Optional description in *Markdown*
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Pet'
          text/plain:
            schema:
              type: string
      responses:
        '201':
          description: Created

  /avatar:
    put:
      summary: Upload an avatar
      requestBody:
        content:
          image/*:
            schema:
              type: string
              format: binary
          multipart/form-data:
            schema:
              type: object
              properties:
                filename:
                  type: array
                  items:
                    type: string
                    format: binary
      responses:
        '201':
          description: Updated

components:
  schemas:
    Pet:
      type: object
      properties:
        name:
          type: string
        fav_number:
          type: integer
      required:
        - name
        - fav_number
      additionalProperties: false

