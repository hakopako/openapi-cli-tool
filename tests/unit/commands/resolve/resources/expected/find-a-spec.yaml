- summary: Add a new pet
  requestBody:
    description: Optional description in *Markdown*
    required: true
    content:
      application/json:
        schema:
          type: object
          properties:
            name:
              type: string
            fav_number:
              type: integer
          required:
            - name
            - fav_number
          additionalProperties: false
      text/plain:
        schema:
          type: string
  responses:
    '201':
      description: Created

